scripts/dependency_install_functions.sh

```diff
@@ -91,7 +91,7 @@ install_cmake() {
     # Mark cmake as installed
     touch "${DATAFED_DEPENDENCIES_INSTALL_PATH}/.cmake_installed-${DATAFED_CMAKE_VERSION}"
   fi
-  export PATH="${DATAFED_DEPENDENCIES_INSTALL_PATH}/bin:${PATH}"
+  export PATH="${DATAFED_DEPENDENCIES_INSTALL_PATH}/bin:$PATH"
```

Comment:
Why did you change this? I don't think this would be the source of the runc problem.

Reply:
Last time we ran into an issue with the tail of the problem `stat /bin/bash: no such file or directory`, the issue was related to PATH being overwritten like in #1125 when building DataFed. I'm otherwise unsure how `bash` would ever be missing.